### PR TITLE
feat(alearm246): create HOW_TO_USE_STRAPI guide

### PR DESCRIPTION
**Pull request by alearm246**

### CODE REVIEW

File: HOW_TO_USE_STRAPI_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- After that, click the green save button on the top right
+
+# Testing the api
+- I'm going to be using insomnia in this tutorial, but you can use any other REST client
+- If any of the endpoints you want to test are public, then all you need to do is make the request for it
+  normally since it doesn't require any authentication
+- If one of your endpoints is protected by authentication, and you try to make a request to it without signing in as your test user,
+  you'll get a 403 forbidden error ![Forbidden](./images/how-to-use-strapi/forbidden.png).
+- To sign in as your test user, make sure to make a POST request to https://localhost:1337/auth/local
+  with a JSON object with the keys identifier and password then click the purple send button. Identifier is just your username or email
```
Comment (chungthuang) [inline HOW_TO_USE_STRAPI_GUIDE.md:29]: What is keys identifier?

File: HOW_TO_USE_STRAPI_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- If you ever need reusable structure, then you'll need to create a component
+- To create a component, click "+ add another field" in the content-type builder screen![Add Field](./images/how-to-use-strapi/add_field.png)
+- After clicking on that, click on the component button ![Component](./images/how-to-use-strapi/component.png)
+- After that, you can fill out the name of the component.
+- You should also fill out the name of the category.
+- You can click on any random icon
+- There are two types of components
+  - Repeatable Components which are best used for multiple instances(If you need an array use this)
+  - Single Component which are best for grouping fields like full addresses, main information, etc.
+- click on either repeatable component or single component(whichever one fits best for your purposes)
```
Comment (chungthuang) [inline HOW_TO_USE_STRAPI_GUIDE.md:73]: Nit: I think `whichever fits` sounds more natural.

File: HOW_TO_USE_STRAPI_GUIDE.md (hunk 1 above)
Comment (alearm246) [reply HOW_TO_USE_STRAPI_GUIDE.md:29]: I explain later in the paragraph “identifier is just your username and email. I also provide an image as an example. It just doesn’t render in here.

**Review by chungthuang — APPROVED** [review]

**Pull request merged**